src/common/utils.ts

```diff
@@ -13,7 +13,7 @@ function isValidSecret(secret: string): boolean {
   }
 }
 
-function dropsToXrp(drops: string | BigNumber): string {
+function dropsToXrp(drops: BigNumber.Value): string {
```

Comment:
New tests that used a number argument caught this type mismatch. A number is valid in the implementation, although we could still intentionally limit this to string in the typings. 

I'd lean towards making this change, since If a user has a number we're unnecessarily asking them to do more work with something like `String(number)` before passing it to us.

Reply:
I'm confused. 

You've dropped the string argument, which makes this method *less* flexible (not more, as your comment seems to say). 

If you do this, you can remove the `if` statement on line 17. 

Also, this method will be a lot more simple if you just use BigInt, which I think is supported as of Node 10.4. Do you really need to support Node 8?

Reply:
Also, what is `BigNumber.Value` vs `BigNumber`? I'm unfamiliar with the notation.

Reply:
Sorry, I should have clarified the change in my comment:
`string | BigNumberInstance`: What we have today. Accepts a string or an already-created BigNumber object, but not a number.
`BigNumber.Value`: An alias to `string | number | BigNumberInstance`, which is what we accept today but PLUS a number. If you're using JavaScript you can pass a number today, and BigNumber supports this, but our types today artificially limit this. This change removes that artificial limit.

Reply:
Ah, got it. Thanks for explaining.